test/JDBC/expected/BABEL-2844.out

```diff
@@ -235,7 +235,7 @@ execute babel_2844_proc 3;
 go
 ~~START~~
 text
-Query Text: EXEC babel_2844_proc 3
+Query Text: EXEC    babel_2844_proc 3
```

Comment:
This is because of https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/c9c7a4b2012afe7492267faf0b0c1e405e80569f/contrib/babelfishpg_tsql/src/tsqlIface.cpp#L5531 : Because the original keyword was EXECUTE, the EXEC keyword is padded with three space to keep the length , and therewith, the positions of subsequent tokens, the same. This is required for the subsequent rewriting steps.